saleor/core/middleware.py

```diff
@@ -116,3 +118,22 @@ def _plugins_middleware(request):
         return get_response(request)
 
     return _plugins_middleware
+
+
+def jwt_refresh_token_middleware(get_response):
+    def middleware(request):
+        """Append generated refresh_token to response object."""
+        response = get_response(request)
+        jwt_refresh_token = getattr(request, "refresh_token", None)
+        if jwt_refresh_token:
+            expires = datetime.utcnow() + settings.JWT_REFRESH_EXPIRATION_DELTA
+            response.set_cookie(
+                JWT_REFRESH_TOKEN_COOKIE_NAME,
+                jwt_refresh_token,
+                expires=expires,
+                httponly=True,
```

Comment:
Can you add a comment saying it is needed to protect against leaking the token to third-parties when improper CORS is set? So if someone ever goes to that code, doesn't try to drop `httponly` somewhere.

Reply:
Good idea